app/templates/index.html.tmpl

```diff
@@ -84,7 +84,7 @@
         line_col = "#8e39a7"
 
         const data = {
-            labels: ['Running', 'Climbing', 'Chasing', 'Eating', 'Foraging'],
```

Comment:
A note for future development: you should only need to edit this file, then run `make generate` to update the other language files (app linting will fail if you don't)